net-dns/dnscrypt-proxy/metadata.xml

```diff
@@ -1,19 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
-		<email>amesists@gmail.com</email>
```

Comment:
Since this is his only package, please also add:

    Closes: https://bugs.gentoo.org/643642

Reply:
Oh, and given the `metadata.xml` you're using for your other packages, you may also want to remove proxy-maint if you wish to only directly work with Lars.